akka-docs/src/main/paradox/scala/stream/stream-flows-and-basics.md

```diff
@@ -354,4 +354,57 @@ been signalled already – thus the ordering in the case of zipping is defined b
 
 If you find yourself in need of fine grained control over order of emitted elements in fan-in
 scenarios consider using `MergePreferred`, `MergePrioritized` or `GraphStage` – which gives you full control over how the
-merge is performed.
\ No newline at end of file
+merge is performed.
+
+# Actor Materializer Life-cycle
```

Comment:
https://en.wikipedia.org/wiki/Life_cycle suggests life cycle is spelled without a dash (but with a space)... might be oldfashioned though?

Reply:
It seems we're using "lifecycle" elsewhere in the docs, shall I move to using that then?